packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -195,7 +195,7 @@
         },
         {
           "command": "sfdx.force.lightning.lwc.preview",
-          "when": "false"
+          "when": "sfdx:project_opened && resource =~ /.*/lwc/[^/]+(/[^/]+\\.(html|css|js))?$/"
```

Comment:
Is the expectation that a user can trigger a preview from the Command Palette ? If so, this won't work since the resource will be empty and the evaluation will always be `false`

Reply:
I don't understand what you mean.  This line allows the preview command to show in the command palette if editor is currently focused on a valid file in an lwc project.  This causes source uri to be undefined when the command is called but that is why I get the uri from the document currently open in the editor windows in that scenario specifically.  Looking at other packages I thought this common practice.

Reply:
I see what you mean, I wasn't opening any component that why I didn't see it in the command palette. We might want to change the label in the future to reflect it only works when opening the file or add it to the docs somewhere.